lib/vmpooler.rb

```diff
@@ -167,6 +170,8 @@ def self.new_metrics(params)
       Vmpooler::Statsd.new(params[:statsd])
     elsif params[:graphite]
       Vmpooler::Graphite.new(params[:graphite])
+    elsif params[:prometheus]
+      Vmpooler::VmPrometheus.new(params[:prometheus])
```

Comment:
I thought you said you were renaming this to promstats in both class name and file name

Reply:
Yes - its done now.